src/shared/modules/settings/settingsDuck.ts

```diff
@@ -77,7 +77,7 @@ const browserSyncConfig = (host = 'https://auth.neo4j.com') => ({
   logoutUrl: 'https://neo4j-sync.auth0.com/v2/logout',
   firebaseConfig: {
     apiKey: 'AIzaSyA1RwZMBWHxqRGyY3CK60leRkr56H6GHV4',
-    databaseURL: 'https://fiery-heat-7952.firebaseio.com',
+    databaseUrl: 'https://fiery-heat-7952.firebaseio.com',
```

Comment:
are we sure this doesn't cause any problems?

Reply:
Just had a look and this changes what variables the firebase API is intialized with... Not sure if it'll break, but I'm starting to wonder whether the minor code smell of inconsistent casing is worth the effort to look through all uses and make sure nothing breaks.. Up to you if you want to keep it and make sure nothing w/ SSO or firebase breaks i guess :)

Reply:
@OskarDamkjaer - I don't know much about the context of SSO. I think maybe the safest way is to revert the change here? I just search the occurrence of `URL` globally and do the replacement, probably it's not ideal to make so many changes.
The E2E tests are passed by the way.

Reply:
I checked with Firebase APIs, it specifies `databaseURL` in its document. So, it'd be better to go with that. I've reverted this change.
Ref: https://firebase.google.com/docs/database/web/start

Reply:
Hmm. I agree it'd be safest to revert since it doesn't really matter if it's capitalized or not. We don't have any end2end tests for SSO yet either..

Reply:
But that said, either way is (probably) fine. Since you changed the firebase one that looked the most suspicious

Reply:
I'll approve, and leave it to your judgement

Reply:
@OskarDamkjaer - I'll leave it for now as this PR involves many changes. In the future, it could be an experimental thing to tweak the naming and see if we break something or not.